coramin/__init__.py

```diff
@@ -0,0 +1,3 @@
+from coramin import utils
+from coramin import domain_reduction
+from coramin import relaxations
```

Comment:
We should discuss these imports - I think this is a reasonable way to do this, but we should have a quick design discussion.